File: Python3/프로그래머스/0/181839. 주사위 게임 1/주사위 게임 1.py
def solution(a, b):
    if a % 2 == 1 and b % 2 == 1:
        return a**2 + b**2
    return (2 * (a + b)) if a % 2 == 1 or b % 2 == 1 else abs(a - b)

File: C++/프로그래머스/0/181839. 주사위 게임 1/주사위 게임 1.cpp
#include <string>
#include <vector>

using namespace std;

int solution(int a, int b) {
    if (a % 2 == 1 && b % 2 == 1) { //모두 홀수라면
        return a*a + b*b;
    } else if (a % 2 == 1 || b % 2 == 1) { //하나만 홀수라면
        return 2 * (a + b);
    }
    return abs(a - b);
}